osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1,6 +1,26 @@
 {
   "chain_name": "osmosis",
   "assets": [
+	  {
+	  		"description": "The native token of Uptick",
+	  		"denom_units": [{
+	  				"denom": "auptick",
+	  				"exponent": 0
+	  			},
+	  			{
+	  				"denom": "uptick",
+	  				"exponent": 18
+	  			}
+	  		],
+	  		"base": "auptick",
+	  		"name": "Uptick",
+	  		"display": "uptick",
+	  		"symbol": "UPTICK",
+	  		"logo_URIs": {
+	  			"png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/uptick/images/uptick.png"
+	  		},
+	  		"coingecko_id": "uptick"
+	  	},
```

Comment:
you can remove this entry to the osmosis-1.assetlist.json file, as it will be regenerated once the other (zone) file is updated.